public/index.html

```diff
@@ -5,79 +5,108 @@
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
+    </div>
+    <div>
+      <main>
+        <ul>
+          <li>
+            <h1>
+              Innovative
+            </h1>
+            <p>Natural-sounding translations that use the grammatical features of each language..</p>
+          </li>
+          <li>
+            <h1>
+              Powerfull & Extensible
+            </h1>
+            <p>
+              Date, time, number, plural rules support. Full bidirectionality.
+              Any internationalization formatters can be added.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Flexible
+            </h1>
+            <p>
+              Can translate on client-side and server-side. Live language switching,
+              robust error handling.
+              JS, Python, Rust and React implementation.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Complete
+            </h1>
+            <p>
+              Visual localization editor, linters, checkers and
+              workflow tooling.
+            </p>
+          </li>
+        </ul>
+        <aside id="demo"></aside>
+      </main>
+    </div>
+    <div>
+      <section id="explainer">
+        <h1>Why are we doing it?</h1>
+        <p>
+        Software localization has been dominated by an outdated paradigm: the translation is just a dictionary of strings which map one-to-one to the English (en-US) copy. This paradigm is unfair and limiting to languages with grammars more complex than English.
+        <br/>
+        For any grammatical feature not supported by English, a special case must be added to the source code, leaking logic into all translations. Furthermore, creating good UIs which depend on multiple external arguments is hard and requires the developer to understand the grammar of the languages the product targets.
+        </p>
+        <p>
+        The design of Fluent is targeting those limitations and is centered around the following <a href="https://github.com/projectfluent/fluent/wiki/Design-Principles">five concepts</a>: Control and Isolation, Simplicity, Tolerance, Modularity and composability, and The Principle of Least Power.
         </p>
-
-        <a class="link--cta" href="https://github.com/projectfluent">Visit GitHub →</a>
       </section>
-
-      <section class="section">
-        <div class="features">
-          <h2 class="features__title">For localizers</h2>
-          <ul class="features__list">
-            <li><strong class="highlight">Natural-sounding translations</strong>
-            <li>Number and date formatting
-            <li>Plural rules
-            <li>Genders
-            <li>Grammatical cases
+    </div>
+    <div>
+      <footer>
+        <p class="cr">Copyright 2017 Project Fluent</p>
```

Comment:
Shouldn't this say `Copyright 2017 Mozilla`?

Reply:
We could also drop the whole footer completely.

Reply:
Tokyo uses "Tokyo project", I think it's ok to keep it as "Fluent Project" and I doubt there's any legal problem with it.
I'd prefer not to remove it for visual purposes. I also hope that once we get more project presence, we'll add more links next to the Github icon in the footer.